serverless/knative_eventing/serverless-channels.adoc

```diff
@@ -0,0 +1,29 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-channels"]
+= Using Channels
+include::modules/common-attributes.adoc[]
+:context: serverless-channels
+
+toc::[]
+
+It is possible to sink events from an event source to a Knative Eventing channel.
+Channels are Kubernetes Custom Resources (CRs) which define a single event forwarding and persistence layer.
+After events have been sent to a channel, these events can be sent to multiple Knative services by using a subscription.
+
+The default configuration for channel instances is defined in the `default-ch-webhook` ConfigMap, however developers can still create their own channels directly by instantiating a supported channel object.
+
+== Supported channel types
+Currently, {ServerlessProductName} only supports the use of InMemoryChannel channels as part of the Knative Eventing Technology Preview.
+
+[IMPORTANT]
+====
+InMemoryChannel channels are a development only channel and are not suitable for use in production.
```

Comment:
s/development only channel/development-only channel/

Reply:
Good point! It will be necessary when Eventing is GA because InMemoryChannels are basically for development, whereas channels that come later like Kafka won't be, but for now I can remove this.